includes/html/print-alert-rules.php

```diff
@@ -101,26 +101,45 @@
 
 echo '</select></td>';
 
-$query = 'FROM alert_rules';
-$where = '';
-$param = [];
+$query_device = 'select alert_device_map.rule_id as id from alert_device_map';
```

Comment:
Can you UPPERCASE the query so it looks like it's a MySQL query please. I.e `SELECT alert_device_map.rule_id AS id FROM alert_device_map`

Reply:
@laf 
done